### PR TITLE
feat: mod tools prediction create flow

### PR DESCRIPTION
**Pull request by rileymiller**

### CODE REVIEW

File: components/activities/activities-tab/activities-tab.tsx [hunk 1]
```diff
@@ -84,6 +106,13 @@ export function ActivitiesTabManager<
             </div>
           )
           : <div className="empty-list-group">No live activities</div>}
+        {hasPollPermissions
```
Comment (austinhallock) [inline components/activities/activities-tab/activities-tab.tsx:109]: is there a good reason to use ternary over && ?

imo ternary always starts getting a little messy. compared to

```
hasPollPermissions && <Button> ... </Button>
```

File: components/activities/create-activity-dialog/create-activity-dialog.scss.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  flex: 1;
+  height: 100%;
+
+  .body {
+    padding: 0 24px;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (austinhallock) [inline components/activities/create-activity-dialog/create-activity-dialog.scss.js:18]: we should probably figure out a good standardization of when to use grid vs flex

File: components/activities/create-activity-dialog/create-activity-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  const onClick = () => {
+    popDialog();
+    pushPage(<CreatePredictionPage />);
+  };
+
+  return (
+    <CreateActivityTile
+      icon={CRYSTAL_BALL_ICON}
+      iconViewBox={20}
+      color={"#AF7AF2"}
```
Comment (austinhallock) [inline components/activities/create-activity-dialog/create-activity-dialog.tsx:51]: rm {}

File: components/activities/create-activity-dialog/create-activity-dialog.scss.js (hunk 2 above)
Comment (rileymiller) [reply components/activities/create-activity-dialog/create-activity-dialog.scss.js:18]: yeah I agree on needing to standardize, any ideas?

Comment (austinhallock) [reply components/activities/create-activity-dialog/create-activity-dialog.scss.js:18]: the general consensus on line is anything one-directional use flex, 2d use grid

**Pull request merged**